ql/math/generallinearleastsquares.hpp

```diff
@@ -75,7 +75,7 @@ namespace QuantLib {
         void calculate(
```

Comment:
This is in the `protected` section, so in theory it's a breaking change. I'd leave the old version as an overload calling the new one and mark it as deprecated.